### PR TITLE
Update office-365-migration-best-practices.md

### PR DESCRIPTION
**Pull request by TransVaultCTO**
Updated Transvault entry to use new formatting (lower case v) and to expand the narrative to match those of other vendors. Also, removed a duplicated entry for Skykick.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/office-365-migration-best-practices.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For more information about Microsoft 365 or Office 365 networking and performanc
...
...s synchronize mailboxes, public folders and calendar information while maintaining coexistence throughout the migration. <br/><br/> [Transvault](https://www.transvault.com/cloud-office-migrations/email-archive-migrations/): Provider of Cloud Office migration solutions to Microsoft 365 from Exchange and Notes. Transvault supports 23 different sources for migration and has products which deliver any size of project, complex email archive migrations and PST management. The enterprise migration solutions are secure, compliant, efficient and user-focused, and can be run both on-premises and in the Cloud. <br/><br/> [SkyKick](https://www.skykick.com/): Provider of automated migration solutions to move on-premises Exchange, Gmail, POP3, IMAP, Lotus Notes to Microsoft 365 or Office 365. The end-to-end migration tools help partners with the sales, planning, migration, management, and onsite phases of the migration project. <br/><br/> [BCC Collaboration Company](http://www.bcchub.com/): Helping companies by supporting their collaboration migration strategy. Best in class supplier of migration tools based on Domino platform, for migrating to Microsoft Exchange, Microsoft 365, and Office 365.|
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/office-365-migration-best-practices.md:49]:
```suggestion
|Third-party migration|There are many tools available from third parties. They use distinctive protocols and approaches to conduct email migrations from email platforms like IBM Lotus Notes and Novell GroupWise.|Here are some third-party migration tools and partners that can assist with Exchange migrations from third-party platforms: <br/><br/> [Binary Tree](https://binarytree.com): Provider of cross-platform messaging migration and coexistence software, with products that provide for the analysis of and the coexistence and migration between on-premises and online enterprise messaging and collaboration environments based on IBM Lotus Notes and Domino and Exchange and SharePoint. <br/><br/> [BitTitan](https://www.bittitan.com): Provider of migration solutions to Microsoft 365 or Office 365. <br/><br/> [CodeTwo](https://www.codetwo.com/): Provider of Microsoft 365 and Office 365 migration solutions. Migrate mailboxes and public folders from any on-premises Exchange and email from IMAP (Gmail, IBM Notes, etc.) to the cloud or perform tenant to tenant migrations. Transfer all data at once or in batches and automate the entire migration process, with zero downtime. <br/><br/> [Quadrotech](https://www.quadrotech-it.com): Provider of migration solutions to Microsoft 365 or Office 365. <br/><br/> [Quest](https://www.quest.com/solutions/office-365/): Provider of migration solutions from Exchange, Microsoft 365, Office 365, Gmail, GroupWise, Notes, POP/IMAP, Zimbra, Sun ONE/iPlanet, PSTs and email archives to Microsoft 365 or Office 365 and Exchange. Quest solutions synchronize mailboxes, public folders, and calendar information while maintaining coexistence throughout the migration. <br/><br/> [Transvault](https://www.transvault.com/cloud-office-migrations/email-archive-migrations/): Provider of cloud office migration solutions to Microsoft 365 from Exchange and Notes. Transvault supports 23 different sources for migration and has products that deliver any size of project, complex email archive migrations, and PST management. The enterprise migration solutions are secure, compliant, efficient, and user-focused, and can be run both on-premises and in the cloud. <br/><br/> [SkyKick](https://www.skykick.com/): Provider of automated migration solutions to move on-premises Exchange, Gmail, POP3, IMAP, Lotus Notes to Microsoft 365 or Office 365. The end-to-end migration tools help partners with the sales, planning, migration, management, and onsite phases of the migration project. <br/><br/> [BCC Collaboration Company](http://www.bcchub.com/): Helping companies by supporting their collaboration migration strategy. Best in class supplier of migration tools based on Domino platform, for migrating to Microsoft Exchange, Microsoft 365, and Office 365.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/office-365-migration-best-practices.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/mailbox-migration/office-365-migration-best-practices.md:49]: @TransVaultCTO - please commit minor edits as indicated.

**Pull request merged**